test/e2e-go/features/transactions/onlineStatusChange_test.go

```diff
@@ -168,12 +169,20 @@ func TestCloseOnError(t *testing.T) {
 	// get the current round for partkey creation
 	_, curRound := fixture.GetBalanceAndRound(initiallyOnline)
 
+	tempDir := os.TempDir()
```

Comment:
I think that we'll need a bit more than that : 
```
The directory is neither guaranteed to exist nor have accessible permissions.
```

Reply:
Changed to:
```
	tempDir, err := ioutil.TempDir(os.TempDir(), "test-close-on-error")
	require.NoError(t, err)
	defer os.RemoveAll(tempDir)
```